doc/build/changelog/unreleased_14/4428.rst

```diff
@@ -0,0 +1,8 @@
+.. change::
+    :tags: bug, orm
+    :tickets: 4428
```

Comment:
**mike bayer** (zzzeek) wrote:

if we want to backport this change to 1.3, then this file should go in the unreleased_13/ folder instead.   however the backport is not a straight cherry-pick as the mechanics of join() have changed.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2099